### ISSUE TITLE
Feature: Improve volume keyboard interaction

### ISSUE THREAD

**Issue by sampotts** [videojs/v10#1132]
It should be possible to adjust volume via:

- Keyboard arrow keys when the mute/unmute button is focused.
- Mouse wheel when the mute/unmute button is hovered.
